src/components/Settings/SettingsMain.tsx

```diff
@@ -219,6 +226,28 @@ const SettingsMain: React.FC = () => {
                     )}
                   </div>
                 </div>
+                <div className="form-row">
+                  <label htmlFor="region" className="text-label">
+                    {intl.formatMessage(messages.region)}
+                  </label>
+                  <div className="form-input">
+                    <div className="flex max-w-lg rounded-md shadow-sm">
+                      <Field as="select" id="region" name="region">
+                        <option value="">
+                          {intl.formatMessage(messages.regionDefault)}
+                        </option>
+                        {regions?.map((region: Region) => (
+                          <option
+                            key={region.iso_3166_1}
+                            value={region.iso_3166_1}
+                          >
+                            {region.english_name}
```

Comment:
We will want to display the region names in whatever language the user currently has set.

Reply:
Unfortunately, I don't think that TMDB will return different values for this. We have the same issue with `Original Language` displayed on Movie/TV detail pages.

Reply:
Yeah, the region endpoint only seems to return the names in English unfortunately.

Reply:
Ah, that's super unfortunate.  Guess we can't really do anything about this.